uniflow-android-test/src/main/java/io/uniflow/android/test/SimpleTestObservers.kt

```diff
@@ -36,7 +35,14 @@ class TestViewObserver {
 
     @Deprecated("better use verifySequence")
     fun assertReceived(vararg any: UIData) = verifySequence(*any)
-    fun verifySequence(vararg any: UIData) = assert(this.values == any.toList()) { "Wrong values\nshould have [${any.toList()}]\nbut was [${values}]" }
+    fun verifySequence(vararg testingData: UIData) {
+        val testingValues = testingData.toList()
+        values.forEachIndexed { index, uiData ->
+            assert(uiData == testingValues[index]) {
+                "Wrong values at [$index] - Type: ${uiData::class.simpleName}\nShould have [${testingValues[index]}]\n  but was [${uiData}]"
+            }
```

Comment:
Prefer `assertEquals(expected, actual, message)` from package `kotlin.test`.

Reply:
yep, assert is also from Kotlin no? 🤔

Reply:
`assert` is a JVM thing. And you have to enable assertions for them to work. By default, assertions are disabled!

- https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/assert.html

Reply:
👍